course_discovery/settings/base.py

```diff
@@ -638,3 +638,6 @@
 DISCOVERY_BASE_URL = "http://localhost:18381"
 
 PRODUCT_API_URL = ''
+
+# Required for communicating with Google Service Account
+GOOGLE_SERVICE_ACCOUNT_CREDENTIALS = '{"key":"value"}'
```

Comment:
```suggestion
GOOGLE_SERVICE_ACCOUNT_CREDENTIALS = {"key":"value"}
```

In production.py we are getting value from vault so we are parsing fine in production.py but locally we are expecting value as a dict. So we can replace this variable to dict value